package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ml-testing-toolkit-ui",
-  "version": "11.1.0",
+  "version": "11.1.1",
```

```suggestion
  "version": "11.2.0",
```

Since this is a new feature?